alphadia/libtransform.py

```diff
@@ -298,14 +298,7 @@ def forward(self, input: SpecLibBase) -> SpecLibBase:
 
         input.charged_frag_types = charged_frag_types
 
-        # Check if CPU or GPU/MPS should be used
-        device = "cpu"
-        if self.use_gpu:
-            try:
-                device = "mps" if os.uname().sysname == "Darwin" else "gpu"
-            except AttributeError:
-                # Windows does not support uname
-                device = "gpu"
+        device = utils.get_torch_device(self.use_gpu)
```

Comment:
if os is Darwin, "mps" is also possibly not available.

Reply:
Very true, I thought this might be handled by peptdeep but I've included it now.